test/dca/strategies/wbtc/wbtc.test.ts

```diff
@@ -0,0 +1,33 @@
+import { ethers } from "hardhat"
+import { deployUUPSUpgradeableContract } from "../../../../scripts/helper/contract"
+import { getTokenContract } from "../../../../scripts/helper/helper"
+import { currentBlockchainTime } from "../Strategy.test"
+import { depositUsdcFarmBtcConfig } from "./config/depositUsdcFarmBtcConfig"
+
+getTokenContract
+depositUsdcFarmBtcConfig
+
+// todo: uncomment after the contract size is shrinked
+//testDcaStrategy("WbtcFarm - usdc -> farm btc DCA Strategy", deployWbtcDcaStrategy, [], depositUsdcFarmBtcConfig())
+
```

Comment:
I think we need to add upgrade test if we're planning any upgrade.

Reply:
yes, and many other things will need to be added